File: Administrativo/spring-erp-servicio/net/royal/spring/sistema/dominio/filtro/FiltroUsuario.cs
using net.royal.spring.framework.core.dominio;
using System;
using System.Collections.Generic;
using System.Text;

namespace net.royal.spring.rrhh.dominio.dto
{
    public class FiltroUsuario
    {
       public string codUsuario { get; set; }
       public String nombre{ get; set; }

        public ParametroPaginacionGenerico paginacion { get; set; }

        public FiltroUsuario()
        {
            paginacion = new ParametroPaginacionGenerico();
        }

    }
}

File: Administrativo/spring-erp-web/SpringNg/src/app/erp_module/shared/usuario/dominio/FiltroUsuario.ts
import { ParametroPaginacionGenerico } from "../../dominio/ParametroPaginacionGenerico";



export class FiltroUsuario {

    constructor() {
        this.paginacion = new ParametroPaginacionGenerico();
    }
    paginacion: ParametroPaginacionGenerico;

    codUsuario: string;
    nombre: string;

    
}
